google/cloud/google_cloud_cpp_common_unit_tests.bzl

```diff
@@ -1,4 +1,5 @@
 # DO NOT EDIT -- GENERATED BY CMake -- Change the CMakeLists.txt file if needed
```

Comment:
Please consider adding the standard copyright license header to these generated files since they live in the source repo.

Reply:
I think I can fix that too, though the years get annoying.

Reply:
Choose a year that is the first time the files were generated, and set an env var to hardcode it to that year, this is fine. We typically don't update years for files in-place after file creation.